compiler/shared/test/diff/Defunctionalize/OldMonoList.mls

```diff
@@ -42,17 +42,23 @@ fun add2(x) = x+2
 //│   fun apply$Lambda1$2$3 = (this, x,) => +(x, 1,)
 //│   Code(List(main$$5()))
 //│ }
-//│ /!!!\ Uncaught error: java.lang.Exception: Internal Error: Symbol already set for obj
-//│ 	at: mlscript.utils.package$.lastWords(package.scala:227)
-//│ 	at: mlscript.VarImpl.symbol_$eq(helpers.scala:870)
-//│ 	at: mlscript.VarImpl.symbol_$eq$(helpers.scala:864)
-//│ 	at: mlscript.Var.symbol_$eq(syntax.scala:67)
-//│ 	at: mlscript.ucs.Desugarer$VarOps.resolveTermSymbol(Desugarer.scala:124)
-//│ 	at: mlscript.ucs.Desugarer$VarOps.withResolvedTermSymbol(Desugarer.scala:129)
-//│ 	at: mlscript.ucs.stages.Desugaring.desugarPatternSplit(Desugaring.scala:432)
-//│ 	at: mlscript.ucs.stages.Desugaring.$anonfun$desugarTermBranch$2(Desugaring.scala:101)
-//│ 	at: mlscript.pretyper.Traceable.trace(Traceable.scala:39)
-//│ 	at: mlscript.pretyper.Traceable.trace$(Traceable.scala:36)
+//│ ╔══[ERROR] the `if` expression has already been desugared
```

Comment:
This should still be an internal error (with a more helpful error message), not a user-facing type error! The hint doesn't make any sense for a user.